code/obj/cement.dm

```diff
@@ -6,7 +6,9 @@
 	density = 0
 	layer = OBJ_LAYER + 0.9
 	event_handler_flags = USE_CANPASS
-	var/health = 4 //how many hits to destroy + 1
+	var/const/initial_health = 4
```

Comment:
i don't think this is a good value, as this is extremely weak compared to everything else on station.... see ex_act(3) throwing it to -40 or whatever